cmd/earth/main.go

```diff
@@ -1450,6 +1470,41 @@ func (app *earthApp) newBuildkitdClient(ctx context.Context, opts ...client.Clie
 	return bkClient, nil
 }
 
+func (app *earthApp) collectAnalytics(exitCode int, realtime time.Duration) {
+	if app.cfg.Global.DisableAnalytics {
+		return
+	}
+	installID, ci := os.LookupEnv("CI")
+	if !ci {
+		var err error
+		installID, err = getInstallID()
+		if err != nil {
+			installID = "unknown"
+		}
+	}
+	segmentClient := analytics.New("RtwJaMBswcW3CNMZ7Ops79dV6lEZqsXf")
+	segmentClient.Enqueue(analytics.Track{
+		Event:  app.commandName,
```

Comment:
Also add something like `"earth "` or `"CLI "` in front? These events will be mixed in together with UI / website events in the analytics.